mpp/pshmem.h4

```diff
@@ -22,7 +22,7 @@ include(shmem_bind_c.m4)dnl
 #ifndef PSHMEM_H
 #define PSHMEM_H
 
-#include <shmem.h>
+#include <shmem-def.h>
 #include <shmemx.h>
```

Comment:
@jdinan Just a thought, should we keep `shmemx.h` in pshmem? Or should we create `shmemx-def.h` and use it in the same way? At this point, we have only a couple of data types to put there, but may become handy when we have newer ones for 1.5?

Reply:
Sure, this will help to simplify `pshmem.h` a bit more.